docs/pages/custom-rendering.rst

```diff
@@ -25,6 +25,45 @@ a hook that allows arbitrary attributes to be added to the ``<table>`` tag.
     >>> # renders to something like this:
     '<table class="mytable">...'
 
+Also every column gets a class attribute, which by default is the same as the
+column's label. Also, by default, odd rows' class is ``odd`` and even rows'
+class is ``even``. So rows of the ``SimpleTable()`` from previous example
+in django-tables2 default configuration will look like:
+
+.. sourcecode:: html
+
+    <tr class="odd">
+      <td class="id">...</td>
+      <td class="age">...</td>
+    </tr>
+    <tr class="even">
+      <td class="id">...</td>
+      <td class="age">...</td>
+    </tr>
+
+
+You can also specify ``attrs`` attribute when creating a column. django-tables2
```

Comment:
This is a relevant notion but maybe better to link to :ref:`column-attributes` and maybe adjust the documentation over there?
Also note I just added a fix for #241, so make sure to rebase on master before attempting to edit the docs there.